include/z64camera.h

```diff
@@ -1567,7 +1567,7 @@ typedef struct Camera {
     /* 0x0D8 */ f32 xzSpeed;
     /* 0x0DC */ f32 dist;
     /* 0x0E0 */ f32 speedRatio;
-    /* 0x0E4 */ Vec3f posOffset;
+    /* 0x0E4 */ Vec3f playerToAtOffset;
```

Comment:
"player to at" reads really weird, but its just the reality of the term "at" that is used for this thing lol. so I dont really have any suggestions and will just live with it